traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/PartnerOfferingProduct.java

```diff
@@ -0,0 +1,54 @@
+package com.guestlogix.travelercorekit.models;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.List;
+
+public class PartnerOfferingProduct implements Product {
+    private String id;
+    private String title;
+    private Price price;
+    private List<ProductOffering> productOfferings;
```

Comment:
why doesn't this contain `List<PartnerOffering> offerings;` instead? If the goal is to separate them correctly later, better not to use ProductOffering here

Reply:
we are using product offering as passes here . partner offering does not contain the passes.